### PR TITLE
fix subscription docs

### LINKED ISSUE #153: Documentation issue for `subscription` directive
**Describe the bug**
The documentation for `subscription` directive ([here](https://neo4j.com/docs/graphql/current/directives/schema-configuration/type-configuration/#_subscription)) seems not to be up to date.
At least with `@neo4j/graphql@5.4.2` the field within the directive to determine the several `SubscriptionFields` is not called `operations` but `events`

### PR DESCRIPTION
**Pull request by rsill-neo4j**
closes: #153

### CODE REVIEW

**Comment by rsill-neo4j** [comment]
should this also be fixed for older versions?

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-graphql-154.surge.sh

Updated pages:
- [Type configuration](https://neo4j-docs-graphql-154.surge.sh/graphql/5/directives/schema-configuration/type-configuration)

**Comment by mjfwebb** [comment]
> should this also be fixed for older versions?

That would be great if possible

File: modules/ROOT/pages/directives/schema-configuration/type-configuration.adoc [hunk 1]
```diff
@@ -122,15 +122,15 @@ enum SubscriptionFields {
     DELETE_RELATIONSHIP
 }
 
-directive @subscription(operations: [SubscriptionFields!]! = [CREATE, UPDATE, DELETE, CREATE_RELATIONSHIP, DELETE_RELATIONSHIP]) on OBJECT | SCHEMA
+directive @subscription(events: [SubscriptionFields!]! = [CREATE, UPDATE, DELETE, CREATE_RELATIONSHIP, DELETE_RELATIONSHIP]) on OBJECT | SCHEMA
```
Comment (darrellwarde) [inline modules/ROOT/pages/directives/schema-configuration/type-configuration.adoc]: I just checked and it's actually like this, who would have known... 🥲 

```suggestion
directive @subscription(events: [SubscriptionFields!]! = [CREATED, UPDATED, DELETED, RELATIONSHIP_CREATED, RELATIONSHIP_DELETED]) on OBJECT | SCHEMA
```

**Review by darrellwarde — DISMISSED** [review]
It looks good, actually an additional change needed though!

**Comment by darrellwarde** [comment]
> > should this also be fixed for older versions?
> 
> That would be great if possible

I don't know exactly when we made this change though.

**Comment by darrellwarde** [comment]
I just checked - it was changed in version 4, so those docs can also be updated!

File: modules/ROOT/pages/directives/schema-configuration/type-configuration.adoc [hunk 2]
```diff
@@ -139,7 +139,7 @@ type Movie @subscription(operations: [])  {
 .Enable only _movieCreated_ subscription for _Movie_
 [source, graphql, indent=0]
 ----
-type Movie @subscription(operations: [CREATE]) { 
+type Movie @subscription(events: [CREATE]) {
```
Comment (rsill-neo4j) [inline modules/ROOT/pages/directives/schema-configuration/type-configuration.adoc]:
```suggestion
type Movie @subscription(events: [CREATED]) { 
```

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Review by mjfwebb — APPROVED** [review]

**Comment by rsill-neo4j** [comment]
also applied this to 6.x and 4.x

**Pull request merged**